esp8266/Makefile

```diff
@@ -148,7 +148,7 @@ DRIVERS_SRC_C = $(addprefix drivers/,\
 SRC_S = \
 	gchelper.s \
 
-FROZEN_MPY_PY_FILES := $(shell find -L $(FROZEN_MPY_DIR) -type f -name '*.py')
+FROZEN_MPY_PY_FILES := $(shell find -L $(FROZEN_MPY_DIR) -type f -name '*.py') modules/upip.py modules/upip_utarfile.py
```

Comment:
@stinos : Perhaps I'm asking too complex questions, but let me try again. Userstory: a random Window users downloads a git package, and clones a repository containing symlinks. What would they get in place of symlinks? Possible answers: a) symlinks; b) copies; c) nothing; d) clone would error out; e) unknown.

Though well, the general approach here is: a change maker smelled a possible issue, and called the Windows maintainer for consult. The Windows maintainer didn't NAK the change, so it's good to go, and if a problem will pop up later, it won't be a problem of the project, just a problem of the Windows maintainer ;-).